zeppelin-web-angular/projects/zeppelin-sdk/src/interfaces/message-operator.interface.ts

```diff
@@ -160,7 +160,7 @@ export enum OP {
   /**
    * [c-s]
    * ask paragraph editor setting
-   * @param magic magic keyword written in paragraph
+   * @param paragraphText magic keyword written in paragraph
```

Comment:
Please improve this part of the documentation.
My suggestion:
```
@param paragraphId paragraph id
@param paragraphText text of the paragraph
```

Reply:
ok, done

Reply:
Can you please also add the documentation for the first parameter `paragraphId`.